GOVERNANCE.md

```diff
@@ -0,0 +1,127 @@
+# Release Working Group
+
+The Node.js Release Working Group (WG) maintains oversight
+over the Node.js Release, Long Term Support (LTS) and
+Canary in the Gold Mine (CitGM) teams. It manages the release
+and long term support schedule and policies for all Node.js releases.
+
+The WG has final authority over Releases including:
+
+* Release process and tools.
+* Content for all releases.
+* Schedule for all releases.
```

Comment:
I think we should delegate it to the Release WG completely or not at all. "subject to CTC approval" creates more questions than it answers.